addons/website/static/src/scss/website.backend.scss

```diff
@@ -86,44 +86,51 @@
     display: contents;
 }
 
+.o_theme_preview_form_view .o_control_panel { 
+    .o_control_panel_breadcrumbs, .o_control_panel_actions {
+        min-width: auto !important;
+    }
+}
+
```

Comment:
Is this actually needed? There is already a `min-width: 200px` sets to the breadcrumb and actions component